CLM/clm_sensitivity.Rmd

```diff
@@ -0,0 +1,177 @@
+---
+title: "CLM Sensitivity Analysis"
+author: "Kristina Riemer"
+output: github_document
+---
+
+```{r, message=FALSE}
+library(broom)
+library(magrittr)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(cowplot)
+```
+
+Data are parameter values with corresponding response variable values. 
+
+```{r, message=FALSE}
+sens_data <- readr::read_csv("clm5_Equ_C4g_default_ens_e500_output.txt", 
+                                 skip = 1)
+colnames(sens_data) <- gsub(" ", "_", colnames(sens_data))
+sens_data_ex <- sens_data %>% 
+  select(1:10)
+```
+### Example of Sensitivity Analysis for Single Output Variable
+
+#### 1. Elasticity
+
+Sensitivity is the change in the response variable given a change in an input variable, which can be a parameter, initial condition, etc. This is representated by the derivative dY/dX. This is calculated below from the slope of the linear regression between the parameter values and response variable. 
```

Comment:
```suggestion
Sensitivity is the change in the response variable given a change in an input variable, which can be a parameter, initial condition, etc. This is representated by the derivative dY/dX. This is approximated below as the slope of the linear regression between the parameter values and response variable. This is a quick-and-dirty variant of the [`get.sensitivity`](https://github.com/PecanProject/pecan/blob/develop/modules/uncertainty/R/sensitivity.analysis.R#L73) function in PEcAn.




  
```

Reply:
In the future I think it will be worthwhile to see if the PEcAn functions will work out of the box, either installing the PEcAn.uncertainty package or seeing if `source('https://raw.githubusercontent.com/PecanProject/pecan/develop/modules/uncertainty/R/sensitivity.analysis.R')` will work ...

Reply:
Created a new issue for using Pecan's functions, see [#69](https://github.com/danforthcenter/sentinel-detection/issues/69).